### ISSUE TITLE
False positive: CKV_K8S_14 - Pulling images from their digest should be allowed

### ISSUE THREAD

**Issue by JPLachance** [bridgecrewio/checkov#535]
**Describe the bug**
Greetings!

The check CKV_K8S_14 "Image Tag should be fixed - not latest or blank" is failing when we pull a container image from the image digest. Pulling an image from its digest is valid and should be allowed by CKV_K8S_14.

**To Reproduce**
1. Create a simple Kubernetes deployment YAML file
  ```YAML
  apiVersion: apps/v1
  kind: Deployment
  metadata:
    name: nginx-deployment
    labels:
      app: nginx
  spec:
    replicas: 3
    selector:
      matchLabels:
        app: nginx
    template:
      metadata:
        labels:
          app: nginx
      spec:
        containers:
          - name: nginx
            image: 'nginx@sha256:b0ad43f7ee5edbc0effbc14645ae7055e21bc1973aee5150745632a24a752661'
            ports:
              - containerPort: 80
```
2. Run the ckeckov command on your file
3. See the issue:
  ```SHELL
  Check: CKV_K8S_14: "Image Tag should be fixed - not latest or blank"
          FAILED for resource: Deployment.nginx-deployment.default (container 0) - nginx
          File: test.yaml:18-21
          Guide: https://docs.bridgecrew.io/docs/bc_k8s_13

                  18 |         - name: nginx
                  19 |           image: 'nginx@sha256:b0ad43f7ee5edbc0effbc14645ae7055e21bc1973aee5150745632a24a752661'
                  20 |           ports:
                  21 |             - containerPort: 80
  ```

**Expected behavior**
Pulling an image from its digest should not make the CKV_K8S_14 check fail.

**Desktop (please complete the following information):**
 - OS: macOS 10.15.6
 - Python: 3.8.5
 - Checkov Version: 1.0.517

**Additional context**
https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier

**Comment by JPLachance** [comment]
Hello!

After looking closely at Checkov checks for Kubernetes, I noticed we have both CKV_K8S_14 and CKV_K8S_43.

CKV_K8S_14:
> Image Tag should be fixed - not latest or blank

CKV_K8S_43:
> Image should use digest

So I'll suppress CKV_K8S_14 and use only CKV_K8S_43 😄
